service/src/main/kotlin/fi/espoo/evaka/reservations/ReservationControllerCitizen.kt

```diff
@@ -272,34 +277,40 @@ SELECT
     ch.last_name,
     ch.preferred_name,
     ci.id AS image_id,
-    p.placement_min_start,
-    p.placement_max_end,
+    jsonb_agg(
+            jsonb_build_object(
+                    'start', p.placement_start,
+                    'end', p.placement_end
+                ) ORDER BY p.placement_start ASC
+        ) AS placements,
```

Comment:
Could the `jsonb_agg` be moved inside the subquery? That would remove the need of another `GROUP BY` in the outmost query.